06_gravity/06_gravity.jl

```diff
@@ -257,11 +257,11 @@ begin
 end
 
 # ╔═╡ 9769da36-1a19-11eb-21ec-63f612b19380
-md"As a second obtion, we can propose a *Gaussian distribution* instead of a uniforn distribution for *g*, like the one shown below, with a mean of 5 and a variance of 2, and let the model update its beliefs with the points we have."
+md"As a second option, we can propose a *Gaussian distribution* instead of a uniform distribution for *g*, like the one shown below, with a mean of 6 and a variance of 2, and let the model update its beliefs with the points we have."
 
 # ╔═╡ bb570a5e-1a19-11eb-26ca-0dd1a523dc5d
 begin
-plot(Normal(5,2), legend=false, fill=(0, .5,:lightblue))
+plot(Normal(6,2), legend=false, fill=(0, .5,:lightblue))
```

Comment:
@pfcrowe out of curiosity, any reason why you changed the mean to 6?

Reply:
sure, line 260 in the text refers to mean of 6 and likewise a Normal(6,2) is defined at line 278. 

this is similar to the change at 326 (because of line 312)